control-plane-operator/controllers/hostedcontrolplane/render/manifests.go

```diff
@@ -64,6 +64,7 @@ func (c *clusterManifestContext) setupManifests() {
 	c.userManifestsBootstrapper()
 	c.machineConfigServer()
 	c.ignitionConfigs()
+	c.clusterMachineApprover()
```

Comment:
The Machine Approver operate with CAPI resources. These are not HCP versioned. Instead they have independent release cycles. The particular version for capi, autoscaler and machine approver should dictated and reconciled by the Hypershift HostedCluster controller. See
https://github.com/openshift/hypershift/blob/main/hypershift-operator/controllers/hostedcluster/hostedcluster_controller.go#L78-L84

Reply:
sure, that makes sense. will move the approver into the hostedcluster controller. 👍

Reply:
updated to move everything into the hostedcluster controller now 👍